webserver/static/js/feverscreen.ts

```diff
@@ -755,6 +907,124 @@ window.onload = async function() {
     return dest;
   }
 
+  function add_pixel(dest: Float32Array, x: number, y: number, amount: number, width: number, height: number) {
```

Comment:
camelCase not snake_case, here and a few other places

Reply:
Fixed in 3121d8c